install_config/cluster_metrics.adoc

```diff
@@ -52,6 +52,17 @@ endif::[]
 
 == Before You Begin
 
+[WARNING]
+====
+If your OpenShift installation was originally performed on a version previous to
+v1.0.8, even if it has since been updated to a newer version, you will need to 
```

Comment:
Hi, i'm not @adellape but anyway...

If you `git grep ifdef` you can see a bunch of examples.  A specific example that shows the syntax would be [admin_guide/router.adoc, Disabling Statistics View](https://github.com/openshift/openshift-docs/blob/master/admin_guide/router.adoc).